packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react';
+import { useAccountHistoryChart } from '@notional-finance/notionable-hooks';
+import { FiatSymbols } from '@notional-finance/core-entities';
+import { useFiat } from '@notional-finance/notionable-hooks';
+import {
+  THEME_VARIANTS,
+  SECONDS_IN_MONTH,
+  getNowSeconds,
+} from '@notional-finance/util';
+import { colors } from '@notional-finance/styles';
+import { useThemeVariant } from '@notional-finance/notionable-hooks';
+import { FormattedMessage } from 'react-intl';
+
+export const useTotalsChart = () => {
+  const baseCurrency = useFiat();
+  const [secondsMultiple, setSecondsMultiple] = useState(1.5);
+
+  window.addEventListener('resize', () => {
+    if (window.innerWidth <= 1152 && secondsMultiple !== 1.2) {
+      setSecondsMultiple(1.2);
+    }
+    if (window.innerWidth > 1200 && secondsMultiple !== 1.5) {
+      setSecondsMultiple(1.5);
+    }
+  });
+
+  const historyData = useAccountHistoryChart(
+    getNowSeconds() - SECONDS_IN_MONTH * secondsMultiple
```

Comment:
Can you remove all the default parameters set in the useAccountHistoryChart hook and just pass them all directly here? That way we don't get any hidden surprises later.